### DISCUSSION TITLE
web iPhone XS Max 动画加载不出来（IOS 版本 17.0.3）

### DISCUSSION THREAD

**Discussion by liangzi-aha** [Tencent/libpag#1789] [Q&A] [answered]
web iPhone XS Max 动画加载不出来（IOS 版本 17.0.3）

警告
wasm streaming compile failed: TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.

falling back to ArrayBuffer instantiation

**Comment by liangzi-aha** [comment]
手机重启一次就好了，展示不知道什么原因

**Comment by domchen** [comment, marked as answer]
应该是网络问题，加载失败或者加载到损坏的文件。

**Reply by yangchongduo** [reply]
怎么可能，如果是这种问题，怎么可能在这里问

**Comment by ligeisme** [comment]
我也是一样的问题，感觉是ios对运行内存的保护机制，禁用了底层的渲染api，以前遇到过ios echarts图多画几个画不出来的问题
